build.gradle.kts

```diff
@@ -59,4 +60,35 @@ tasks.withType<org.jetbrains.dokka.gradle.DokkaTask>().configureEach {
 }
 tasks.withType<Test> {
   maxParallelForks = (Runtime.getRuntime().availableProcessors() / 2).takeIf { it > 0 } ?: 1
+}
+
+apiValidation {
+  /**
+   * Packages that are excluded from public API dumps even if they
+   * contain public API.
+   */
+//  ignoredPackages.add("kotlinx.coroutines.internal")
+
+  /**
+   * Sub-projects that are excluded from API validation
+   */
+  ignoredProjects.addAll(listOf("extension-style-app", "android-auto-app", "app"))
+
+  /**
+   * Classes (fully qualified) that are excluded from public API dumps even if they
+   * contain public API.
+   */
+//  ignoredClasses.add("com.company.BuildConfig")
+
+  /**
+   * Set of annotations that exclude API from being public.
+   * Typically, it is all kinds of `@InternalApi` annotations that mark
+   * effectively private API that cannot be actually private for technical reasons.
+   */
+//  nonPublicMarkers.add("com.mapbox.maps.MapboxExperimental")
```

Comment:
we could skip the classes/functions annotated with `com.mapbox.maps.MapboxExperimental` so that we ignore changes to these methods. but on the other hand, it's good to keep track of these as they are part of the public interface.

Reply:
maybe better to add one more annotation 
`MapboxInternalApi` or `InternalApi` 

https://github.com/Kotlin/binary-compatibility-validator/issues/36

Reply:
I mean `internal` as access modifier.
Smth that we can change without braking public API